src/main/java/io/codearte/jfairy/producer/person/locale/es/EsNationalIdentityCardNumberProvider.java

```diff
@@ -6,16 +6,18 @@
 import java.util.regex.Pattern;
 
 /**
+ * Spanish National Identity Card Number (known as Documento Nacional de Identidad or DNI)
```

Comment:
@OlgaMaciaszek 
DNI is commented here. Is this not enuogh? Not sure where else you ment it should be commented.